CONTRIBUTING.adoc

```diff
@@ -141,10 +141,10 @@ To facilitate rapid development we've put together a Vagrantfile you can use to
 7.  Now change into the directory with the OpenShift binaries, and start the OpenShift server:
 
         $ cd _output/local/bin/linux/amd64
-        $ sudo ./openshift start --public-master=localhost --volume-dir=</absolute/path> &> openshift.log &
+        $ sudo ./openshift start --public-master=localhost --volume-dir=/home/vagrant/volumes &> openshift.log &
```

Comment:
Add `mkdir /home/vagrant/volumes` somewhere above since this directory doesn't exist by default.